webpack/react_app/redux/actions/TargetingHosts/index.js

```diff
@@ -0,0 +1,12 @@
+import API from 'foremanReact/API';
+
+import { TARGETING_HOSTS_SUCCESS, TARGETING_HOSTS_ERROR } from '../../consts';
+
+export const getTargetingHostsAction = apiUrl => async dispatch => {
+  try {
+    const result = await API.get(apiUrl);
```

Comment:
can you use the middleware for this? `import { get } from 'foremanReact/redux/API'`